website/blog/_guide/YYYY-MM-DD.blog-template.md

```diff
@@ -14,13 +14,13 @@ tags: [tag1, tag2]
 draft: true
 ---
 
+![Image alt description](/img/blog/banner-image.jpg) <!---Please add the path for the image banner (i.e. /img/blog/banner-image.png).-->
```

Comment:
Our recent blogs have image first then description, if that's the way we want let's update the template too :hugs: